docs/proto-docs.md

```
@@ -2400,6 +2403,36 @@ MsgMarketSettleResponse is a response message for the MarketSettle endpoint.
 
+<a name="provenance-exchange-v1-MsgMarketTransferCommitmentsRequest"></a>
+
+### MsgMarketTransferCommitmentsRequest
+MsgMarketTransferCommitmentsRequest is a request message for the MarketTransferCommitments endpoint.
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| `admin` | [string](#string) |  | admin is the account with "cancel" permission requesting this transfer. |
+| `account` | [string](#string) |  | account is the Bech32 address string of the account receiving the transferred funds. |
+| `amount` | [cosmos.base.v1beta1.Coin](#cosmos-base-v1beta1-Coin) | repeated | amount to transfer. |
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)